### PR TITLE
ECO-1116 report blocked restore

### PR DESCRIPTION
**Pull request by liorama**
- [x] linked to a JIRA issue
- [x] description of the change
- [ ] tests for the newly added code
- [ ] if this is a breaking change, contains backward support code
- [ ] if this requires client changes or partner updates, updated dependendants
- [ ] were metrics and alerts added
- [ ] should I change documentation

### CODE REVIEW

File: scripts/src/public/services/users.ts [hunk 1]
```diff
@@ -14,6 +14,7 @@ import { create as createUserRegistrationRequested } from "../../analytics/event
 import { create as createUserRegistrationSucceeded } from "../../analytics/events/user_registration_succeeded";
 import { create as createUserLogoutServerRequested } from "../../analytics/events/user_logout_server_requested";
 import { create as createWalletAddressUpdateSucceeded } from "../../analytics/events/wallet_address_update_succeeded";
+import { create as createRestoreRequestFailed } from "../../analytics/events/restore_request_failed";
```
Comment (nitzantomer) [inline scripts/src/public/services/users.ts]: where's this file?
it's not part of the PR, and i don't see it in master

**Review by nitzantomer — CHANGES_REQUESTED** [review]

File: scripts/src/public/services/users.ts (hunk 1 above)
Comment (liorama) [reply scripts/src/public/services/users.ts]: Yeah, adding the file will benefit the PR immensely

Comment (liorama) [reply scripts/src/public/services/users.ts]: @nitzantomer thanks

**Review by nitzantomer — DISMISSED** [review]

File: scripts/src/analytics/events/restore_request_failed.ts [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+	event_type: "log";
+	common: Common;
+	error_reason: string;
+}
+
+export function create(user_id: string, device_id: string, error_reason: string): Event<RestoreRequestFailed> {
+	return new Event<RestoreRequestFailed>({
+		event_name: "restore_request_failed",
+		event_type: "log",
+		common: createCommon(user_id, device_id),
```
Comment (liorama) [inline scripts/src/analytics/events/restore_request_failed.ts]: @nitzantomer is this ok?

**Review by nitzantomer — APPROVED** [review]
